### ISSUE TITLE
Super slow (x200) exiting after 'require(...)'

### ISSUE THREAD

**Issue by mysticatea** [atom/electron#4886]
* Electron version: v0.37.2
* Operating system: Windows 7 64bit

Probably related in #4265.
I have started investigating since `child_process.fork` was really slow.

```js
// test.js
require("xlsx")
```

```
> node test.js
( about 0.2 seconds )

> set ELECTRON_RUN_AS_NODE=1 && node_modules\.bin\electron test.js
( about 40 seconds )
```

Just importing a module is super slow. 
It seems to wait for unintentional timers.
All modules are not slow, but there are some slow modules: `eslint`, `babel-core`, etc...

Sorry, I'm not sure what difference is between normal modules and slow modules.

**Comment by baconface** [comment]
The usage of require is used in the Node context and not the browser context. So you are likely loading in all the optional modules which are rather large as documented at https://www.npmjs.com/package/xlsx#optional-modules. This is likely the cause of your slow import.

**Comment by mysticatea** [comment]
Actually, import itself is not slow. But after import, the process will take over 30 seconds to exit.
This behavior is happend on `require("eslint")` and `require("babel-core")`.

**Comment by zcbenz** [comment]
This has been fixed by #4927, which will be included in the next release.

**Comment by mysticatea** [comment]
Thank you very much!
